KubeArmor/common/common.go

```diff
@@ -23,6 +23,9 @@ import (
 // == Common == //
 // ============ //
 
+// IsK8SEnv used for specifying k8s environment
+var IsK8SEnv bool
+
```

Comment:
```suggestion
```
Remove this var and use GlobalCfg.K8sEnv bool in the check below.

Reply:
It is not possible to include config in common as it causes circular dependency. Hence this global variable.